scripts/dts/extract/globals.py

```diff
@@ -276,6 +297,14 @@ def enable_old_alias_names(enable):
     global old_alias_names
     old_alias_names = enable
 
+def add_compat_alias(node_address, label_postfix, label, prop_aliases):
```

Comment:
Using 'alias' in the function name is a bit confusing as it suggests to the reader that the function has something to do with DTS aliases node.